### DISCUSSION TITLE
Kotlin API bridge

### DISCUSSION THREAD

**Discussion by A00826925** [Sinytra/ConnectorExtras#13] [Ideas]
compatibility with fabric mods that need "fabric kotlin api"

**Comment by Su5eD** [comment]
What is "fabric kotlin api"?

**Comment by A00826925** [comment]
https://modrinth.com/mod/fabric-language-kotlin

Enables the use of kotlin programming in mods, however it has a forge and fabric versions, idk if its possible to make fabric mods work using the forge version using a bridge
